features/step_definitions/navigation_steps.rb

```diff
@@ -1,3 +1,24 @@
-Given(/^I am on the landing page$/) do
-  visit root_path
-end
\ No newline at end of file
+Given(/^I am on the "([^"]*)" page$/) do |page|
+  case page.downcase
+    when 'landing'
+      path = root_path
+    when 'login'
+      path = new_user_session_path
+    when 'edit my application'
+      path = edit_membership_application_path(@user.membership_applications.last)
+    else
+      path = 'no path set'
+  end
+  visit path
+end
+
+When(/^I try to visit the "([^"]*)" page$/) do |page|
+  case page.downcase
+    when 'applications index'
+      path = membership_applications_path
+    else
+      path = 'path not set'
+  end
+  visit path
+  expect(current_path).not_to be path
+end
```

Comment:
What is the use case for this step? I don't understad and would love some clarification.